readme.md

```diff
@@ -1,24 +1,24 @@
-# text [![License](https://img.shields.io/github/license/KyoriPowered/text.svg)](https://github.com/KyoriPowered/text/blob/master/license.txt) [![Build Status](https://travis-ci.org/KyoriPowered/text.svg?branch=master)](https://travis-ci.org/KyoriPowered/text)
+# adventure [![License](https://img.shields.io/github/license/KyoriPowered/adventure.svg)](https://github.com/KyoriPowered/adventure/blob/master/license.txt) [![Build Status](https://travis-ci.org/KyoriPowered/adventure.svg?branch=master)](https://travis-ci.org/KyoriPowered/adventure)
 
-A text library for Minecraft.
+A serverside user interface library for Minecraft.
```

Comment:
Not technically limited to a server.

Reply:
Minecraft: Java Edition

Reply:
what i was going for is this is a library that doesn't require installation on a client -- since "user interface library" sounds like something for client mods -- thoughts on other phrasing that carries that meaning through?